src/birthplace.js

```diff
@@ -116,7 +116,7 @@ function isMalaysia(code) {
 }
 
 function isForeign(code) {
-    return countryCodePairs[code] != undefined;
+    return countryCodePairs[code] != undefined && !isMalaysia(code);
```

Comment:
This additional check is redundant as there is no case where `isMalaysia` and `isForeign` overlaps. Refer below:
```javascript
function parse(code) {
    if (isMalaysia(code)) return parseMalaysia(code);
    if (isForeign(code)) return parseForeign(code);

    throw new Error('Invalid birth place code');
}
```

If this change is required, provide a valid test case that would fail.

Reply:
Could no find escapee for this.
You are right, if no plan to export isForeign function, this change is unnecessary.